src/app/layout.tsx

```diff
@@ -11,9 +10,8 @@ export default function RootLayout({ children }: { children: React.ReactNode })
         head.tsx. Find out more at https://beta.nextjs.org/docs/api-reference/file-conventions/head
       */}
       <head />
-      <body className="flex flex-col justify-between">
+      <body className="flex flex-col justify-between text-base font-splineSans" >
         <ReactQueryWrapper>
-          <Header />
           <main className="flex-1">{children}</main>
```

Comment:
Noticed that the page doesn't fill the entire available height, makes the footer look a little jumpy

Reply:
@MaiaS could you check it out once again, it should be fixed